src/store.ts

```diff
@@ -0,0 +1,16 @@
+import { configureStore, combineReducers } from "@reduxjs/toolkit";
+import { userReducer } from "./User/userReducer";
+
+// Type Safe Reducers - se
+const rootReducer = combineReducers({
+  user: userReducer.reducer,
```

Comment:
do we need to add the other reducers in here?

Reply:
Oops - good catch! Yep. This where each reducer would need to go. I also see I mysteriously stopped typing the comment. I will fix the comment and add the other two reducers